airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -207,7 +207,7 @@ def _format_msg(self, log_line):
         if self.json_format:
             try:
                 # pylint: disable=protected-access
-                return self.formatter._style.format(_ESJsonLogFmt(**log_line.to_dict()))
+                return self.formatter._style.format(_ESJsonLogFmt(self.json_fields, **log_line.to_dict()))
```

Comment:
@millin Thanks for the PR, can you please also add a test case or update an existing test in https://github.com/apache/airflow/blob/master/tests/providers/elasticsearch/log/test_es_task_handler.py

Reply:
Ok, I'll try

Reply:
I have updated the test